docs/release.md

```diff
@@ -0,0 +1,149 @@
+# steps
+
+1. Sync latest master
+1. new branch
+1. Sync the downstream pipeline repo
+1. Checkout release branch
```

Comment:
Checkout release branch (eg: release-v0.7.0)

Reply:
I called my branch `release` I don't know if the naming of local branch really matters